source/_components/sensor.onewire.markdown

```diff
@@ -68,4 +72,7 @@ Configuration variables:
 
 - **names** array (*Optional*): ID and friendly name of your sensors.
 - **mount_dir** (*Optional*): Location of device tree if owfs driver used.
+- **server** (*Optional*): Hostname or ip of the owserver
+- **port** (*Optional*): port of the owserver, used only if `server`option is not empty. Default to `4304`
+- **timeout** (*Optional*): Max time, in second, to read value from the server. Used only if `server`option is not empty. Default to `2`.
```

Comment:
Missing determiner before `value`, suggestion: `to read a value from the server`